Gemfile.lock

```diff
@@ -1,76 +1,58 @@
 PATH
   remote: .
   specs:
-    omniauth-wish (1.0.0)
```

Comment:
just copy / paste holdover?

Reply:
Maybe. I didn't directly change the gemfile. That's the result of bundling after updating the gemspec.